Frontend/src/pages/admin/AdminStudents.vue

```diff
@@ -1,8 +1,38 @@
 <template>
```

Comment:
nit: change the file name to `AdminStudentsPage.vue` so it's more obvious that it's a page when you have it open in an editor. We didn't do this at first but I've started doing it because I think it's more clear